test/unit/expiration_process_test.lua

```diff
@@ -171,3 +171,44 @@ function g.test_broken_process_expired_tuple(cg)
         t.assert_ge(full_scan_counter, 3)
     end)
 end
+
+function g.test_check_tuples_not_expired_by_timestamp(cg)
+    local space = cg.space
+    local space_archive = cg.space_archive
+    local task_name = cg.task_name
+
+    local total = 5
+    for i = 1, total do
+        local time = fiber.time()
```

Comment:
The change will make it a little easier to find a problem in case of a flaky test + it is a logic of the original test.

```suggestion
    local total = 5
    local time = fiber.time()
    for i = 1, total do
```